mitmproxy/web/app.py

```diff
@@ -176,6 +176,7 @@ def get(self):
         sio.close()
 
     def post(self):
+        # self.state.clear()
```

Comment:
Just calling `.clear()` here is the best way for now. We need to fix that later.